_episodes/06-loops-and-functions.md

```diff
@@ -381,8 +381,9 @@ def one_year_csv_writer(this_year, all_data):
     """
     Writes a csv file for data from a given year.
 
-    this_year --- year for which data is extracted
-    all_data --- DataFrame with multi-year data
+    Keyword arguments:
```

Comment:
This lesson does not explain what a "keyword argument" is.

Reply:
IMHO the [_mention_ & example in 06-loops...](https://github.com/datacarpentry/python-ecology-lesson/blob/855e70ceebb567265bba9e3445f87e400a93347c/_episodes/06-loops-and-functions.md#L627-L638) is a sufficient explanation.

Reply:
Hm... a brief note in the entire lesson doesn't not present itself as a sufficient introduction to something new. If you'd like to make the lesson consistent, it might be better to remove that note, if possible.

Reply:
OK, see 4a1bd24.

Reply:
Thank you, Katrin!